modules/minimp3/resource_importer_mp3.cpp

```diff
@@ -113,7 +113,7 @@ Ref<AudioStreamMP3> ResourceImporterMP3::import_mp3(const String &p_path) {
 
 Error ResourceImporterMP3::import(const String &p_source_file, const String &p_save_path, const HashMap<StringName, Variant> &p_options, List<String> *r_platform_variants, List<String> *r_gen_files, Variant *r_metadata) {
 	bool loop = p_options["loop"];
-	float loop_offset = p_options["loop_offset"];
+	double loop_offset = p_options["loop_offset"];
```

Comment:
Could we make these changes in a separate PR since it didn't end up fixing the MP3 pops?

Reply:
For sure, I'll change it back.